docs/testcases.md

```diff
@@ -1,396 +1,35 @@
-# Treeherder Test Cases
+# Treeherder Manual Smoke Tests
 
-These tests should be run prior to deploying code changes to the Treeherder production environment.
+Quick set of tests to be run prior to:
 
-## Initial page load
+- Merging a PR
+- Deploying from [stage](https://treeherder.allizom.org) to [production](https://treeherder.mozilla.org)
 
-Load Treeherder. eg.
+For a PR, depending on what you changed, you may need to only test in one of
+these categories.
 
-- [stage](https://treeherder.allizom.org)
-- [production](https://treeherder.mozilla.org)
+<!-- prettier-ignore -->
+!!! note
+    Open the console to look for exceptions.
 
-Depending on your test requirement.
+## Treeherder
 
-**Expected**: Page loads displaying pushes pushed to mozilla-inbound.
+1. Initial page loads. Can load more jobs with **Next N** buttons.
+2. Switch repos.
+3. Classify a job.
+4. Retrigger a job on try.
+5. Add new jobs and click **Trigger** button on try.
+6. Hit `n` key to select next failed job. Hit again should cycle through each.
+7. Change filtering to ensure they take effect.
+8. Login / Logout
```

Comment:
Another test case:

* Toggle between unclassified jobs and all filtered jobs should work
   * Select an orange job
   * Click on link to filter that kind of job
   * Press letter 'U' to toggle between unclassified jobs and all jobs
     * You should be switching between all jobs of that kind and the orange ones that have not been classified

When I upgraded [react-hot-keys](https://github.com/mozilla/treeherder/commit/b961adf761f8f7140bc297866dde21a5ca866fdf) I faced this issue.

Reply:
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1575928